src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc

```diff
@@ -44,8 +44,10 @@ NaviStartStreamRequest::NaviStartStreamRequest(
     , EventObserver(application_manager.event_dispatcher())
     , retry_number_(0) {
   SDL_AUTO_TRACE();
+  const ApplicationManagerSettings& settings =
+      application_manager_.get_settings();
   std::pair<uint32_t, int32_t> stream_retry =
-      application_manager_.get_settings().start_stream_retry_amount();
+      settings.start_stream_retry_amount();
```

Comment:
@orhan-mehmedov my task is to move applicable existing UTs and create appropriate ones if existant are not applicable. This test is not written by me but I believe new variable is introduced in favor of easier code readability.